src/pages/Models/Details/ModelDetails.js

```diff
@@ -104,6 +106,8 @@ const ModelDetails = () => {
 
   const [viewFilterToggle, setViewFilterToggle] = useState({ all: true });
 
+  const [slidePanelData, setSlidePanelData] = useState({});
```

Comment:
I'm not sure this approach as implemented will work because of the subtle differences between the entity type displays.

Instead of this taking `{ app }` maybe it should instead just take the view type then we have a different component rendered into the panel depending on the type. It would then be this components responsibility to fetch the data it needs from the redux store.

(typescript I know, but it helps to illustrate the data type)
```typescript
enum Entity {
  app = "app",
  unit = "unit",
  machine = "machine",
  relation = "relation",
};

const panelType Entity = "app";

setSlidePanelData({ panelType });
```

Reply:
Noted but, as discussed, will refactor to this approach after this initial app panel lands.